Documentation/CRDs/Shared-Filesystem/ceph-filesystem-crd.md

```diff
@@ -135,13 +135,16 @@ spec:
     # for more details see: https://docs.ceph.com/en/latest/dev/cephfs-mirroring/#bootstrap-peers
     peers:
       secretNames:
-        - secondary-cluster-peer
+      # Add the secret name if it already exists else specify the empty list here.
     # specify the schedule(s) on which snapshots should be taken
     # see the official syntax here https://docs.ceph.com/en/latest/cephfs/snap-schedule/#add-and-remove-schedules
     snapshotSchedules:
       - path: /
         interval: 24h # daily snapshots
-        startTime: 11:55
+        startTime: 2022-07-15T11:55:00
```

Comment:
In our yamls, the comments about settings show before the setting, so please move this down after the new comments.

Reply:
this is not a valid time after some time. I think it makes sense to comment it out.

Reply:
For now I've commented `startTime` if it sounds good.